staging/src/k8s.io/cli-runtime/pkg/printers/interface.go

```diff
@@ -35,3 +36,19 @@ type ResourcePrinter interface {
 	// Print receives a runtime object, formats it and prints it to a writer.
 	PrintObj(runtime.Object, io.Writer) error
 }
+
+// PrintOptions struct defines a struct for various print options
+type PrintOptions struct {
```

Comment:
can we remove the declaration in pkg/printers/interface.go? I don't see any references to the old type

Reply:
Fixed.